src/core/engine/processors/kitchen_simulator.py

```diff
@@ -144,3 +147,29 @@ def _order_send_to_complete(self, order_to_be_complete):
         )
         self.order_manager.add_to_queue(order_to_be_complete)
         print(f"order {order_to_be_complete.id} completed at {datetime.now()}")
+
+    def _init_queues_before_kitchen_start(self):
+
+        orders_in_status_new = self.order_controller.get_orders_by_status(OrderStatus.NEW_ORDER.name, order_limit=500)
+        orders_in_status_in_process = self.order_controller.get_orders_by_status(OrderStatus.IN_PROCESS.name, order_limit=500)
```

Comment:
Put 500 behind a constant.

Be consistent I also see a 1000 not sure if they are the same thing

Reply:
done